repository/repository.js

```diff
@@ -0,0 +1,50 @@
+export class Repository {
+  constructor(Document) {
+    this.Document = Document;
+  }
+  async getItems() {
+    try {
+      const items = await this.Document.find();
+
+      return items;
+    } catch (error) {
+      return error.message;
```

Comment:
In case of success mongoDB query you're returning array of items, but in case of error you are returning a String. That's also valid for the rest of the repository methods.

Instead it would be nice to always have the same data type returned in both cases success & error.
Success:
```
{
  data: items,
  error: null
}
```

Error:
Success:
```
{
  data: null,
  error: error
}
```